.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -115,7 +115,6 @@ kernel_matrix_testing_setup_env:
   stage: kernel_matrix_testing
   rules:
     !reference [ .on_system_probe_changes_or_manual ]
-  allow_failure: true
```

Comment:
does this mean kernel-testing-matrix framework is on a critical path of the dd-agent CI now?

Reply:
It just means that the environment setup job has to always pass. The tests themselves may still fail. This is to ensure that any updates by the e2e squad does not break our system in main.